include/unk_020851B8.h

```diff
@@ -0,0 +1,15 @@
+#ifndef POKEDIAMOND_UNK_020851B8_H
+#define POKEDIAMOND_UNK_020851B8_H
+
+struct UnkStruct_020851B8 {
+    u8 unk00[0xC];
+};
```

Comment:
More likely:
```c
{
  u32 flags[2];
  u8 unk_8;
  u8 unk_9;
  u8 unk_A;
  u8 unk_B;
};
```

Reply:
Seems to match with the 64 berries, I didn't see that, mb